lib/sandbox/sandbox_linux.go

```diff
@@ -58,6 +59,9 @@ type NetNs struct {
 
 // SymlinkCreate creates the necessary symlinks for the NetNs
 func (n *NetNs) SymlinkCreate(name string) error {
+	if n.netNS == nil {
+		return errors.New("no netns set up")
```

Comment:
It would be great if we can use `fmt.Errorf` here to be consistent within the file. The import above can then be removed.